api/design/user.go

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2021 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package design
+
+import (
+	"github.com/tektoncd/hub/api/design/types"
+	. "goa.design/goa/v3/dsl"
+)
+
+var _ = Service("user", func() {
+	Description("The user service exposes endpoint to get user specific specs")
+
+	Error("invalid-token", ErrorResult, "Invalid User token")
+	Error("invalid-scopes", ErrorResult, "Invalid User scope")
```

Comment:
I don't think we have used this error in the implementation of the api

Reply:
It is being by the auth service while verifying jwt